src/connectedk8s/azext_connectedk8s/vendored_sdks/operations/connected_cluster_operations.py

```diff
@@ -500,9 +515,7 @@ def internal_paging(next_link=None, raw=False):
             response = self._client.send(request, stream=False, **operation_config)
 
             if response.status_code not in [200]:
-                exp = CloudError(response)
-                exp.request_id = response.headers.get('x-ms-request-id')
-                raise exp
+                raise models.ErrorResponseException(self._deserialize, response)
```

Comment:
how is the error response experience in this case? Is it the same as before?

Reply:
Will track these things, when we test in staging in next iteration.